packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.1.0
```

Comment:
I believe we've marked this as `# 4.1.0 (Unreleased)` in the passed, so we know quickly where to add additional items.

I think this is all we need to do for now, until we decide to do a components release.

Reply:
Updated!